server/indexers/github_helpers.py

```diff
@@ -0,0 +1,122 @@
+from universalclient import Client, jsonFilter
+from server import settings
+import subprocess
+from server import schemas
+import json
+import bs4
+import re
+
+es_client = Client(settings.ES_HOST, dataFilter=jsonFilter)
+search_client = es_client.search
+ac_client = es_client.autocomplete
+history_client = es_client.history
+
+
+def save_indexed_version(gh_type, repo_name, typ, version):
+    doc_id = (gh_type + '/' + repo_name).replace('/', '%2F')
+    # TODO: create; update if error
+    resp = history_client._(typ)._(doc_id)._update.post(data={'version': version})
+    if resp.status_code == 500:
+        history_client._(typ)._(doc_id).put(data={'version': version})
```

Comment:
One concern I have about saving the version is that it's decoupled from the bulk_data save and I don't think it has to be.  Do you save the new version first, or the data? either way there's potential to get out of sync if there's an error in between.  Not a huge deal, but would it be better if save_indexed_version instead returned the json code to insert the data?